docs/pages/web/daterange.js

```diff
@@ -0,0 +1,333 @@
+
+When there’s not a date range selected, the call-to-action disables to prevent user errors.
+          `}
+        >
+          <MainSection.Card
+            cardSize="md"
+            sandpackExample={
+              <SandpackExample
+                code={implementation}
+                name="implementation example"
+                previewHeight={500}
+              />
+            }
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="With RadioGroup"
+          description="This variant allow users to select common options of date ranges, so they can select an option without having to navigate through the calendar picker."
+        >
+          <MainSection.Card
+            cardSize="md"
+            title="Future selection"
+            sandpackExample={
+              <SandpackExample
+                code={futureRadiogroup}
+                name="future radiogroup"
+                previewHeight={500}
+              />
+            }
+          />
+          <MainSection.Card
+            cardSize="md"
+            title="Past selection"
+            sandpackExample={
+              <SandpackExample code={pastRadiogroup} name="past radiogroup" previewHeight={500} />
+            }
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Error messaging"
+          description={`
+DateRange can communicate errors when the user selects an invalid date. Use \`startDateErrorMessage\`, \`endDateErrorMessage\` and \`onStartDateChange\`, \`onEndDateChange\`. \`onStartDateBlur\`, \`onStartDateFocus\`, \`onEndDateBlur\`, \`onEndDateFocus\` to implement error messaging correctly.
+
+The following implementation shows how to use all required props for error messaging.
+          `}
+        >
+          <MainSection.Card
+            cardSize="md"
+            sandpackExample={
+              <SandpackExample code={errorMessaging} name="error example" previewHeight={500} />
+            }
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Disable future & past"
+          description="DateField supports disabling future and past dates from being selected."
+        >
+          <MainSection.Card
+            cardSize="md"
+            title="Disable past"
+            sandpackExample={
+              <SandpackExample code={disabledPast} name="past example" previewHeight={500} />
+            }
+          />
+          <MainSection.Card
+            cardSize="md"
+            title="Disable future"
+            sandpackExample={
+              <SandpackExample code={disabledFuture} name="future example" previewHeight={500} />
+            }
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Supporting locales"
+          description="Select the correct locale to see DataRange's selected locale support."
```

Comment:
*Daterange

I think could be rephrased as:

> Daterange supports multiple locales. Use the dropdown to try out different locales by passing in the `localeData` prop.

If it's going to be in the variant section, I think it might be helpful to have a code example

Reply:
DONE first part. 

If we decide to implement date-fns in the codesandboxes sure. If I do example, it would break atm.

We can do that as a separate project for all cmps using data-fns